processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -30,17 +30,17 @@
 {
   private final VSizeIndexedInts column;
   private final VSizeIndexed multiValueColumn;
-  private final GenericIndexed<String> lookups;
+  private final CachingIndexed<String> cachedLookups;
 
   public SimpleDictionaryEncodedColumn(
       VSizeIndexedInts singleValueColumn,
       VSizeIndexed multiValueColumn,
-      GenericIndexed<String> lookups
+      CachingIndexed<String> cachedLookups
```

Comment:
I agree, but I would rather fix that as part of a separate PR rather than changing the existing behavior